### PR TITLE
Another pass over documentation

### PR DESCRIPTION
**Pull request by bart-degreed**
Minor corrections and cleanup of documentation.

### CODE REVIEW

**Comment by bart-degreed** [comment]
@fdlane Can you review these please?

File: docs/usage/extensibility/layer-overview.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ public void ConfigureServices(IServiceCollection services)
     services.AddResourceService<FooService>();
 
     // custom repository
-    services.AddScoped<AFooRepository>();
+    services.AddResourceRepository<FooRepository>();
```
Comment (fdlane) [inline docs/usage/extensibility/layer-overview.md:38]: Is `AddResourceRepository` a thing yet?

File: docs/usage/sparse-field-selection.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Sparse Field Selection
 
-As an alternative to returning all attributes from a resource, the fields query parameter can be used to select only a subset.
+As an alternative to returning all attributes from a resource, the fields query string parameter can be used to select only a subset.
```
Comment (fdlane) [inline docs/usage/sparse-field-selection.md:3]: Would it be easier to read with some markdown love "...the `fields` query string parameter"

File: docs/usage/toc.md [hunk 3]
```diff
@@ -4,15 +4,15 @@
...
 
 # [Resource Graph](resource-graph.md)
-# [Metadata](meta.md)
-# [Filtering](filtering.md)
 # [Options](options.md)
-# [Errors](errors.md)
-# [Including Relationships](including-relationships.md)
-# [Pagination](pagination.md)
-# [Routing](routing.md)
+# [Filtering](filtering.md)
```
Comment (fdlane) [inline docs/usage/toc.md:8]: Was it intentional to not have the TOC in alpha order?

Comment (fdlane) [reply docs/usage/toc.md:8]: @bart-degreed ...Note that this review only sent you one email. :) Hope it is better this time.  As always, thank you for all the work.

File: docs/usage/extensibility/layer-overview.md (hunk 1 above)
Comment (bart-degreed) [reply docs/usage/extensibility/layer-overview.md]: No it is not, I thought I might have accidentally deleted it, but it was never there. I've reverted this change.

File: docs/usage/sparse-field-selection.md (hunk 2 above)
Comment (bart-degreed) [reply docs/usage/sparse-field-selection.md]: Good point, changed.

File: docs/usage/toc.md (hunk 3 above)
Comment (bart-degreed) [reply docs/usage/toc.md:8]: It wasn't ordered alphabetically before, I changed based on logical order and what was in the old documentation at the root.

Yes, one email indeed! :)

**Pull request merged**